src/components/form.svelte

```diff
@@ -1,98 +1,137 @@
 <script lang="ts">
   import { bugStore } from '$lib/store';
+  import { page } from '$app/stores';
 
   import {
     Button,
     Input,
-    FloatingSelect,
     Radio,
     Label,
+    FileInput,
+    Select,
     FileUpload,
     type FileUploadItem
   } from '@significa/svelte-ui';
   import { enhance, type SubmitFunction } from '$app/forms';
 
   let teams = $bugStore.teams;
+  let author = $bugStore.userName;
 
   let files: FileUploadItem[] = [];
   let attachments = '';
   $: if (files.length > 0) {
     attachments = files
       .filter((f) => f.status === 'success')
       .map((f) => f.url)
-      .join(',');
+      .join(', ');
   } else {
     attachments = '';
   }
 
   let priorityType = 'low' || 'high' || 'medium' || 'critical';
   const priorities = [
     {
-      name: 'Low',
+      name: 'low',
       description:
         'A non-urgent bug, this bug does not affect core functionality of the product'
     },
     {
-      name: 'Medium',
+      name: 'medium',
       description:
         'This bug affects functionality but on a non-core user journey.'
     },
     {
-      name: 'High',
+      name: 'high',
       description:
         'This bug is causing core-functionality problems but not breaking the product.'
     },
     {
-      name: 'Critical',
+      name: 'critical',
       description: 'The product can not function with this bug.'
     }
   ];
+  const prioritiesRequest = [
+    {
+      name: 'low',
+      description: 'A non-urgent request, to be tackled whenever theres time.'
+    },
+    {
+      name: 'medium',
+      description: 'This request is important but on a non-core user journey.'
+    },
+    {
+      name: 'high',
+      description:
+        'This is a core-functionality request that needs to be tackled as soon as possible.'
+    },
+    {
+      name: 'critical',
+      description: 'The product can not function without this.'
+    }
+  ];
 
   $: selectedType = 'bug';
-  let error = false;
 
   // To prevent the page to update
   const onSubmit: SubmitFunction = (input) => {
-    console.log(input);
+    //console.log(input);
   };
 </script>
 
 <form action="?/submitReport" method="POST" use:enhance={onSubmit}>
+  <input type="hidden" name="author" bind:value={author} />
```

Comment:
since this is a hidden input, I'm wondering if you really want to bind the value or just default the value as `author`?
Instead you can do `bind:value={$bugStore.username}` or without the bind `value={$bugStore.username}`